packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.html

```diff
@@ -15,6 +15,7 @@
     <a
         class="oui-link oui-link_icon"
         data-ng-href="{{:: $ctrl.datastoreOrderLink }}"
+        data-ng-click="$ctrl.trackClick($ctrl.ORDER_HOST_TRACKING_PREFIX)"
```

Comment:
Can you use the tracking directives here ? (data-track-type, data-track-name and data-track-on)